exe/Main.hs

```diff
@@ -30,6 +30,7 @@ import Development.IDE.Types.Logger
 import Development.IDE.Plugin
 import Development.IDE.Plugin.Completions as Completions
 import Development.IDE.Plugin.CodeAction as CodeAction
+import Development.IDE.Plugin.Test as Test
```

Comment:
Wonder if we can move the plugin to the test directory, and use a cabal flag to enable it only for tests.

Reply:
We don't build ghcide in the tests stanza, it's only built once in the main stanza. But I've changed Main to "link" the plugin only when `--testing` is given